docs/pom.xml

```diff
@@ -40,11 +40,96 @@
                         </goals>
                         <configuration>
                             <target>
-                                <exec dir="${project.basedir}" executable="jekyll" failonerror="true">
+                                <available file="${project.build.directory}/che-repo" property="cheRepoExists" type="dir"/>
+                                <mkdir dir="${project.build.directory}/che-repo"/>
+                                <exec dir="${project.basedir}" executable="docker" failonerror="true">
+                                    <arg value="run" />
+                                    <arg value="--name" />
+                                    <arg value="jekyll_git_che" />
+                                    <arg value="jekyll/jekyll" />
+                                    <arg value="sh" />
+                                    <arg value="-c" />
+                                    <arg value="cd /srv/jekyll/ ; git init ; git remote add -t add-docs-images -f origin https://github.com/eclipse/che.git ; git checkout add-docs-images" />
```

Comment:
version of che should be specified.
It's useful when we have to perform bugfixes release because in that case we don't use master branch of che inside codenvy